src/Modules/Profile/Commands.Profile/InvokePowerBIRestMethod.cs

```diff
@@ -203,6 +204,9 @@ protected virtual void PopulateClient(IAccessToken token, HttpClient client)
             client.BaseAddress = new Uri(this.Profile.Environment.GlobalServiceEndpoint);
             client.DefaultRequestHeaders.Accept.Clear();
             client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Bearer", token.AccessToken);
+            client.DefaultRequestHeaders.UserAgent.Clear();
+            client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("MicrosoftPowerBIMgmt-InvokeRest", PowerBICmdlet.CmdletVersion));
```

Comment:
You may want to declare "MicrosoftPowerBIMgmt-InvokeRest" as a constant somewhere to avoid duplicating this string.

Reply:
This is the only place its used and don't foresee it being used elsewhere.  MicrosoftPowerBIMgmt is duplicated but even moving it to a constant isn't going to fully remove duplication as it lives in non-CS files and is used in filenames. So I figured if it was going to be renamed it will need a find-replace approach.